src/openforms/authentication/types.py

```diff
@@ -125,3 +125,37 @@ class YiviContext(TypedDict):
         "unknown",
     ]
     authorizee: YiviAuthorizee
+
+
+class EIDASNaturalPersonSubject(TypedDict):
+    identifierType: Literal["bsn", "national_id", "opaque"]
+    identifier: str
+    firstName: str
+    familyName: str
+    dateOfBirth: str
+
+
+class EIDASCompanySubject(TypedDict):
+    identifierType: Literal["opaque"]
+    identifier: str
+    companyName: str
+
+
+class EIDASNaturalPersonAuthorizee(TypedDict):
+    legalSubject: EIDASNaturalPersonSubject
+
+
+class EIDASCompanyAuthorizee(TypedDict):
+    legalSubject: EIDASCompanySubject
+    actingSubject: EIDASNaturalPersonSubject
+
+
+class EIDASContext(TypedDict):
+    source: Literal["eidas"]
+    levelOfAssurance: Literal[
+        "urn:oasis:names:tc:SAML:2.0:ac:classes:PasswordProtectedTransport",
+        "urn:oasis:names:tc:SAML:2.0:ac:classes:MobileTwoFactorContract",
+        "urn:oasis:names:tc:SAML:2.0:ac:classes:Smartcard",
+        "urn:oasis:names:tc:SAML:2.0:ac:classes:SmartcardPKI",
+    ]
```

Comment:
I'm not sure about this, in particular when logging in as natural person vs. company - these LOA sets don't overlap.